runfolder/handlers.py

```diff
@@ -51,22 +67,23 @@ def get(self, path):
 
     def post(self, path):
         """
-        Sets the state of the runfolder
+        Sets the state of the runfolder. TODO: Not implemented
         """
-        self.runfolder_svc.set_runfolder_state(path, "TODO")
+        # self.runfolder_svc.set_runfolder_state(path, "TODO")
+        raise NotImplementedError()
 
     @arteria.undocumented
     def put(self, path):
         """
         NOTE: put is provided for test purposes only.
-
-        TODO: Discuss if it should be disabled in production
         """
+        # TODO: Discuss if it should be disabled in production
```

Comment:
I think it definitely should be enabled in debug mode, since it makes full integration tests easy, but yes, I don't think it shouldn't be available in production either.

So I'll add a check if in debug mode if there are no other suggestions.